### PR TITLE
BugFixes

### PR DESCRIPTION
**Pull request by iosdevrashad**
Scaled our views for different screen sizes adhering to different iphone and ipad versions.

### CODE REVIEW

**Comment by Eugeneberezin** [comment]
Whenever we do UI changes I’d appreciate if we could see the screenshots with different screen sizes. But it’s ok for now.

File: Food4Soul/Food4Soul/Views/ContentView.swift [hunk 1]
```diff
@@ -11,6 +11,23 @@ struct ContentView: View {
...
+            return 310
+        } else if DeviceTypes.isiPad {
+            return 400
+        } else if DeviceTypes.isiPhoneX {
+            return 360
+        }
+        else {
+            return 380
+        }
+
```
Comment (Eugeneberezin) [inline Food4Soul/Food4Soul/Views/ContentView.swift:28]: Extra space

File: Food4Soul/Food4Soul/Views/MainRecipiesView.swift [hunk 2]
```diff
@@ -138,7 +138,7 @@ struct MainRecipiesView: View {
             image: "Greens")
     ]
     
-    let colums = [GridItem(.adaptive(minimum: 200), spacing: 5), GridItem(.adaptive(minimum: 100), spacing: 5)]
+    let colums = [GridItem(.adaptive(minimum: 200), spacing: 5), GridItem(.adaptive(minimum: 200), spacing: 5)]
```
Comment (Eugeneberezin) [inline Food4Soul/Food4Soul/Views/MainRecipiesView.swift:141]: NICE!

**Review by Eugeneberezin — APPROVED** [review]

File: Food4Soul/Food4Soul/Views/ContentView.swift (hunk 1 above)
Comment (iosdevrashad) [reply Food4Soul/Food4Soul/Views/ContentView.swift:28]: I will do this in the future. Thanks for the advice.

**Comment by iosdevrashad** [comment]
> Whenever we do UI changes I’d appreciate if we could see the screenshots with different screen sizes. But it’s ok for now.

I will do this in the future. Thanks for the heads up.

**Review by Eugeneberezin — APPROVED** [review]

**Pull request merged**